### DISCUSSION TITLE
Booting PINN from SSH

### DISCUSSION THREAD

**Discussion by Pazulay** [procount/pinn#687] [Q&A]
Hello!

So I bought a Pi 4 because I have to use it in a Computer Science class but I want to start using it privately as well.

For school we have to use a modified version of Debian Buster OS to access network and user settings.

But at home I would like to use it as a retro emulator.

I just found out about PINN and think it could help with my problem.

My issue is that the OS we use at school is headless and we solely control it via ssh from our PCs. We just plug our Pies into the outlet and access it with SSH. We don't even have any keyboards or monitors to access a GUI.

So I wanna know if there is a way to boot PINN in similar fashion so when I am at school I can access the PINN boot via ssh and select to boot the School's Debian OS.

**Comment by procount** [comment]
Why don't you just use 2 SD cards - one for school and one for home?
